src/main/java/info/tehnut/soulshards/core/mixin/MobSpawnerLogicEntityId.java

```diff
@@ -0,0 +1,15 @@
+package info.tehnut.soulshards.core.mixin;
+
+import com.sun.istack.internal.Nullable;
+import net.minecraft.util.Identifier;
+import net.minecraft.world.MobSpawnerLogic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+@Mixin(MobSpawnerLogic.class)
+public interface MobSpawnerLogicEntityId
+{
+    @Nullable
```

Comment:
Remove this nullable. It's not the javax one since I don't pull that lib

Reply:
Fixed